qiskit/runner_wrapper.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Helper module for simplified Qiskit usage.
+
+    This module includes
+        run_circuits: runs a list of quantum circuits.
+        run_schedule: (TODO) runs a schedule of pulses.
```

Comment:
what these TODO means?

Reply:
These functions don't exists. But we should comment. What is the wrapper functions these are the ones i think we should have? I wanted to add this so that we set what the future has comming.